mne/label.py

```diff
@@ -2020,6 +2036,147 @@ def read_labels_from_annot(subject, parc='aparc', hemi='both',
+        if subject is None:
+            subject = label.subject
+        if subject is not None:  # label.subject can be None, depending on init
+            if subject != label.subject:
+                raise ValueError('Got multiple values of %s: %s and %s'
+                                 % (name, subject, label.subject))
+    if subject is None:
+        raise ValueError('if label.subject is None for all labels, '
+                         '%s must be provided' % name)
+    return subject
+
+
+@verbose
+def morph_labels(labels, subject_to, subject_from=None, subjects_dir=None,
+                 surf_name='white', verbose=None):
+    """Morph a set of labels.
+
+    This is useful when morphing a set of non-overlapping labels (such as those
+    obtained with :func:`read_labels_from_annot`) from one subject to
+    another.
+
+    Parameters
+    ----------
+    labels : list
+        The labels to morph.
+    subject_to : str
+        The subject to morph labels to.
+    subject_from : str | None
+        The subject to morph labels from. Can be None if the labels
+        have the ``.subject`` property defined.
+    subjects_dir : string, or None
+        Path to SUBJECTS_DIR if it is not set in the environment.
+    surf_name : str
+        Surface used to obtain vertex locations, e.g., 'white', 'pial'
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see :func:`mne.verbose`
+        and :ref:`Logging documentation <tut_logging>` for more).
+
+    Returns
+    -------
+    labels : list
+        The morphed labels.
+
+    See Also
+    --------
+    read_labels_from_annot
+    mne.Label.morph
+
+    Notes
+    -----
+    This does not use the same algorithm as Freesurfer, so the results
+    morphing (e.g., from ``'fsaverage'`` to your subject) might not match
+    what Freesurfer produces during ``recon-all``.
+
+    .. versionadded:: 0.18
+    """
+    subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
+    subject_from = _check_labels_subject(labels, subject_from, 'subject_from')
+    mmaps = read_morph_map(subject_from, subject_to, subjects_dir)
+    vert_poss = [_load_vert_pos(subject_to, subjects_dir, surf_name, hemi,
+                                mmap.shape[0])
+                 for hemi, mmap in zip(('lh', 'rh'), mmaps)]
+    idxs = [_sparse_argmax(mmap, axis=1) for mmap in mmaps]
+    out_labels = list()
+    values = filename = None
+    for label in labels:
+        li = dict(lh=0, rh=1)[label.hemi]
+        vertices = np.where(np.in1d(idxs[li], label.vertices))[0]
+        pos = vert_poss[li][vertices]
+        out_labels.append(
+            Label(vertices, pos, values, label.hemi, label.comment, label.name,
+                  filename, subject_to, label.color, label.verbose))
+    return out_labels
+
+
+@verbose
+def labels_to_stc(labels, values, tmin=0, tstep=1, subject=None, verbose=None):
+    """Convert a set of labels and values to a STC.
```

Comment:
I don't get what this function is supposed to be doing? why passing tmin and tstep?

Reply:
I'm not sure how to make the description clearer. Basically if you have a set of labels, and a set of values for those labels, it will create a STC from those (with values in the correct vertices based on the labels). Think of it as the opposite of something like `extract_label_time_course`.

For STC constructor you need to provide `tmin` and `tstep`, so they need to be in this function.

Reply:
Maybe adding that it's like the opposite of `extract_label_time_course` would help? I could also add it to `See Also`